java/ql/lib/change-notes/2021-08-10-cleartext-storage-sharedprefs-query.md

```diff
@@ -0,0 +1,4 @@
+---
+category: newQuery
+---
+* The query "Cleartext storage of sensitive information using `SharedPreferences` on Android" (`java/android/cleartext-storage-shared-prefs`) has been promoted from experimental to the main query pack. Its results will now appear by default. This query was originally [submitted as an experimental query by @luchua-bc](https://github.com/github/codeql/pull/4675).
```

Comment:
Should be in `src` dir rather than `lib`.

Reply:
🤦‍♂️ you're right, thanks. Fixed.